tools/run_tests/xds_k8s_test_driver/framework/helpers/skips.py

```diff
@@ -65,6 +65,8 @@ def version_gte(self, another: str) -> bool:
         """
         if self.version == 'master' or self.version is None:
             return True
+        if another == 'master':
+            return False
         return self._parse_version(self.version) >= self._parse_version(another)
 
     def version_lt(self, another: str) -> bool:
```

Comment:
Nit: Can we just remove one of `version_gte` and `version_lt`. Can be done in a separate PR.